packaging/windows/generatemsi.ps1

```diff
@@ -144,18 +213,35 @@ if(-Not (RunCandle))
     Exit -1
 }
 
+if(-Not (RunCandleForBundle))
+{
+    Exit -1
+}
+
 if(-Not (RunLight))
 {
     Exit -1
 }
 
+if(-Not (RunLightForBundle))
+{
+    Exit -1
+}
+
 if(!(Test-Path $DotnetMSIOutput))
 {
     throw "Unable to create the dotnet msi."
     Exit -1
 }
 
+if(!(Test-Path $DotnetBundleOutput))
+{
+    throw "Unable to create the dotnet bundle."
+    Exit -1
+}
+
 Write-Host -ForegroundColor Green "Successfully created dotnet MSI - $DotnetMSIOutput"
```

Comment:
In that case can you publish the bundle instead of the msi @ https://github.com/dotnet/cli/pull/1130/files#diff-17675b33dd200e853ba5a9dd40d4f0e9R249